src/librustc_mir/interpret/operand.rs

```diff
@@ -571,6 +571,19 @@ impl<'a, 'mir, 'tcx, M: Machine<'a, 'mir, 'tcx>> EvalContext<'a, 'mir, 'tcx, M>
         })
     }
 
+    /// This is used by [priroda](https://github.com/oli-obk/priroda) to get an OpTy from a local
+    pub fn read_local_of_frame(
+        &self,
+        frame: &super::Frame<'mir, 'tcx, M::PointerTag>,
+        local: mir::Local,
+        layout: Option<TyLayout<'tcx>>,
```

Comment:
Uh, I forgot we already know the layout there sometimes. Exposing this is not nice.

I do not have a good idea for how to avoid it, though.

Reply:
Layout is also passed to `eval_place_to_op` which calls this function now.